packages/clerk-js/src/core/resources/Session.ts

```diff
@@ -166,7 +168,7 @@ export class Session extends BaseResource implements SessionResource {
     const cachedEntry = skipCache ? undefined : SessionTokenCache.get({ tokenId }, leewayInSeconds);
 
     // Dispatch tokenUpdate only for __session tokens with the session's active organization ID, and not JWT templates
-    const shouldDispatchTokenUpdate = !template && options?.organizationId === Session.clerk.organization?.id;
```

Comment:
One bug: `organizationId` gets a default value, but `options.organizationId` does not, so this check wasn't working as intended 👀